cogs/techinfo.py

```diff
@@ -91,9 +91,9 @@ async def androidusb(self, ctx):
         embed.set_footer(text="Note: EtchDroid will not work for Windows ISOs as they don't follow the ISOHybrid standard.")
         await ctx.send(embed=embed)
 
-    @commands.hybrid_command(description="Get instructions on how to run the system file checker in Windows 10/11")
```

Comment:
Even better would be to just say Windows, afaik these commands work on older versions too.